assets/app/scripts/filters/resources.js

```diff
@@ -412,6 +412,28 @@ angular.module('openshiftConsole')
       return details.id;
     };
   })
+  .filter('isRecentBuild', function(ageLessThanFilter) {
```

Comment:
since we're using it in a directive, I'm ok with it being a filter... maybe `isActiveOrRecentBuild`?